.env.example

```diff
@@ -2,6 +2,10 @@
 BOT_TOKEN=''
 CHANNEL_ID=''
 
+## Logs Configuration
+# accept integer or valid strings as described in the doc: https://docs.python.org/3/library/logging.html#logging.Logger.setLevel
```

Comment:
Are you sure it accepts an integer? Because even if the doc says so (and it works) the env variable **isn't cast (in `main.py`)** to an int so passing `LOG_LEVEL=10` will result in 
```python
discord.utils.setup_logging(root=True, level="10")
```
not
```python
discord.utils.setup_logging(root=True, level=10)
```
(I know this is the code I gave you but it seems I made a mistake ahah)

Reply:
Both cases are now handled so we can be more precise by using integers, or simply use the normalized strings.

Reply:
tks!

Reply:
[9dbbb91](https://github.com/iScsc/iscsc.fr-blog-notify/pull/8/commits/9dbbb91105df0e18b200aded299fa4983b37a575)